scripts/update-pycodestyle.sh

```diff
@@ -67,7 +67,7 @@ done
 
 echo "--- applying isort"
 for SOURCE in $SOURCES; do
-    isort -y $SOURCE
+    isort $SOURCE
```

Comment:
it should have been called before with python3, not sure why there was no errors before

Reply:
Just to confirm, there is no action required here, right?

Thank you!

Reply:
no action, just wondering if you know why there was no error before

Reply:
There were no previous test running `update-pycodestyle.sh` script with python3.x versions.

Reply:
ok, that explains it.